src/components/TodoList/TodoList.jsx

```diff
@@ -1,44 +1,104 @@
 import React from 'react';
 import './TodoList.scss';
+import propTypes from 'prop-types';
+import { Todo } from '../Todo';
 
-export const TodoList = () => (
-  <div className="TodoList">
-    <h2>Todos:</h2>
+export class TodoList extends React.Component {
+  state = {
+    formmatedTodos: [],
+    isRandom: false,
+    statusFilter: 0,
+    searchQuarry: '',
+  }
 
-    <div className="TodoList__list-container">
-      <ul className="TodoList__list">
-        <li className="TodoList__item TodoList__item--unchecked">
-          <label>
-            <input type="checkbox" readOnly />
-            <p>delectus aut autem</p>
-          </label>
+  componentDidMount() {
+    this.setState({ formmatedTodos: [...this.props.todos] });
```

Comment:
no need to save todos in TodoList state, just passed filtered todos to TodoList and use them as props